login.css

```diff
@@ -0,0 +1,126 @@
+body {
+  font-family: 'Lucida Sans Unicode', 'Lucida Sans', 'Lucida Sans Regular',
+    'Lucida Grande', Geneva, Verdana, sans-serif;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("images/bg.png");
+  background-size: cover;
+  
+}
+
+header{
+    position: absolute;
+    top: 0;
+    background-color:purple;
+    left: 0;
+    z-index: 10;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    color: white;
+    padding: 10px 100px 0px;
+    align-items: center;
+}
+
+header nav{
+    display: flex;
+}
+
+header nav ul a{
+  color: white;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+a:hover {
+  color: green;
+}
+
+.login {
+  max-width: 1000px;
+  min-width: 800px;
+  width: 70%;
+  height: 500px;
+  border-radius: 16px;
+  background-image: url("images/bg.png");
+  display: flex;
+  box-shadow: 1px 1px 20px 2px rgba(0, 0, 0, 0.3);
```

Comment:
Great job. 
include this style in the login class. it is going to fill up the page with that background
    background-size: cover;

Reply:
Done